Makefile

```diff
@@ -2,3 +2,12 @@ x11_bsd_flags = -I/usr/X11R6/include -L/usr/X11R6/lib
 
 all:
 	${CC} ${CFLAGS} ${LDFLAGS} clipnotify.c -o clipnotify $(x11_bsd_flags) -lX11 -lXfixes
+
+install: all
+	cp clipnotify /usr/local/bin
+
+uninstall:
+	rm /usr/local/bin/clipnotify
```

Comment:
Some minor nits that I have tried to address in #13, commenting here for completeness:

 - Usually `/usr/local` is not hardcoded by indirectly defined via a `PREFIX` variable, in that way the user or package system can use their favourite `PREFIX`
 - A possible - by default not defined - `DESTDIR` is also often honored. In that way it is possible to perform a staged install. This is also a requirement for several package systems and by using that the possible root requirements are requested only in the real install phase.
 - `/usr/local/bin` or `${PREFIX}/bin` could not exist

Thanks!